jmeter/plans/apply.rb

```diff
@@ -0,0 +1,92 @@
+require 'ruby-jmeter'
+
+BASEURL = ENV.fetch('JMETER_TARGET_BASEURL')
+
+def url(path)
+  BASEURL + path
+end
+
+# 975 concurrent users, ~7 minutes per session
+test do
+  cookies clear_each_iteration: true
+  thread_count = 975
+
+  threads count: thread_count, continue_forever: true, duration: 420 do
+    #-> Sign up
+    visit name: 'Account page', url: url('/candidate/account') do
+      extract name: 'authenticity_token', regex: 'name="authenticity_token" value="(.+?)"'
+    end
+    think_time 2000
+    submit(
+      name: 'Account page - create an account', url: url('/candidate/account'),
+      fill_in: {
+        'candidate_interface_create_account_or_sign_in_form[existing_account]' => 'false',
+        'authenticity_token' => '${authenticity_token}',
+      }
+    )
+    think_time 1000
+    visit name: 'Sign up page', url: url('/candidate/sign-up') do
+      extract name: 'authenticity_token', regex: 'name="authenticity_token" value="(.+?)"'
+    end
+    think_time 2000
+    submit(
+      name: 'Sign up page - submit email', url: url('/candidate/account'),
+      fill_in: {
+        'authenticity_token': '${authenticity_token}',
+        'candidate_interface_sign_up_form[email_address]': '${__threadNum}' + '@loadtest.example.com',
```

Comment:
This uses the current thread number to generate a consistent email address both here and later on when we sign in. It does mean that each subsequent run of the load test will need the candidates table cleared. Is there a better way to do this?